src/TideEntityUpdateHelper.php

```diff
@@ -496,15 +496,15 @@ public function repositionComponentsRelativeToKey(array $components, array $new_
    *  ];
    *  $tide_helper->repositionEntityFormDisplayFields('node.landing_page.default', 'field_topic',['field_show_topic_term_and_tags'=>$field_show_topic_term_and_tags])
    */
-  public function repositionEntityFormDisplayFields(string $entity_form_display_id, string $after_field, array $fields_to_insert): void {
+  public function repositionEntityFormDisplayFields(string $entity_form_display_id, string $anchor_field, array $fields_to_insert, string $position = 'after'): void {
```

Comment:
It's a little thing, but if you made your options for `$position` class fields the IDE could auto complete?

Reply:
yep, Thanks! I will make this change !